controllers/src/test/scala/cromwell/pipeline/controller/UserControllerTest.scala

```diff
@@ -36,16 +36,17 @@ class UserControllerTest
       "return the sequence of users" taggedAs Controller in {
         val usersByEmailRequest: UserEmail = UserEmail("someDomain@mail.com", Enable.Unsafe)
         val dummyUser: User = TestUserUtils.getDummyUser()
+        val dummyUserNoCredentials = UserNoCredentials.fromUser(dummyUser)
         val userId = dummyUser.userId
-        val uEmailRespSeq: Seq[User] = Seq(dummyUser)
+        val uEmailRespSeq: Seq[UserNoCredentials] = Seq(dummyUserNoCredentials)
```

Comment:
Can you please rename current `User` to `UserWithCredentials` and `UserNoCredentials` to `User`? Since user without creds should be our default choice now, it's better to give it a convenient name

Reply:
@myazinn done